pallets/capacity/src/mock.rs

```diff
@@ -0,0 +1,150 @@
+use super::*;
+use crate as pallet_capacity;
+
+use common_primitives::{
+	node::AccountId,
+	schema::{SchemaId, SchemaValidator},
+};
+use frame_support::{
+	parameter_types,
+	traits::{ConstU16, ConstU32, ConstU64},
+};
+use sp_core::{ConstU8, H256};
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, Convert, IdentityLookup},
+	AccountId32,
+};
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+// Configure a mock runtime to test the pallet.
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
+		Msa: pallet_msa::{Pallet, Call, Storage, Event<T>},
+		Capacity: pallet_capacity::{Pallet, Call, Storage, Event<T>},
+	}
+);
+
+impl frame_system::Config for Test {
+	type BaseCallFilter = frame_support::traits::Everything;
+	type BlockWeights = ();
+	type BlockLength = ();
+	type DbWeight = ();
+	type RuntimeOrigin = RuntimeOrigin;
+	type RuntimeCall = RuntimeCall;
+	type Index = u64;
+	type BlockNumber = u64;
```

Comment:
```suggestion
	type BlockNumber = u32;
```
To match the size/type for BlockNumber used in runtime.

We could import `common_primitives::node:BlockNumber`
and then set the line to
`type BlockNumber = BlockNumber;` so that the type for the size of the block number is consistent throughout the project